lib/rules/no-invalid-position-at-import-rule/index.js

```diff
@@ -0,0 +1,67 @@
+// @ts-nocheck
+
+'use strict';
+
+const report = require('../../utils/report');
+const ruleMessages = require('../../utils/ruleMessages');
+const validateOptions = require('../../utils/validateOptions');
+const valueParser = require('postcss-value-parser');
+
+const ruleName = 'no-invalid-position-at-import-rule';
+
+const messages = ruleMessages(ruleName, {
+	rejected: (atImport) => `Unexpected invalid position @import rule ${atImport}`,
```

Comment:
```suggestion
	rejected: () => `Unexpected invalid position @import rule`,
```

In my [original blueprint](https://github.com/stylelint/stylelint/issues/5133#issuecomment-774642535), I made a mistake by including the value of the URL in the violation message. It is, unlike with the `no-duplicates` rule, not relevant here.

Let's remove:

- the value from the violation messages
- the associated code in rule
- the value from the tests

Sorry for causing extra effort in writing the rule.